src/languages/sql_more.js

```diff
@@ -0,0 +1,181 @@
+/*
+ Language: SQL More (mix of MySQL, Oracle, etc)
+ Contributors: Nikolay Lisienko <info@neor.ru>, Heiko August <post@auge8472.de>, Travis Odom <travis.a.odom@gmail.com>, Vadimtro <vadimtro@yahoo.com>, Benjamin Auder <benjamin.auder@gmail.com>
+ Website: https://en.wikipedia.org/wiki/SQL
+ Category: database
+ */
+
+/*
+
+This is a preservation of the old bloated SQL grammar which includes pretty much
+the kitchen sink because no one was keeping track of which keywords belong to
+which databases.  This is likely to be removed in the future.
+
+- Oracle SQL should be factored into it's own 3rd party grammar.
+- MySQL should be factored out into it's own 3rd party grammar.
+
+*/
+
+export default function(hljs) {
+  var COMMENT_MODE = hljs.COMMENT('--', '$');
+  return {
+    name: 'SQL (more)',
+    aliases: ["mysql", "oraclesql"],
```

Comment:
I'd suggest `oracle`, because `MySQL` and `Oracle` are the names of the database systems.